src/lib/components/footer.svelte

```diff
@@ -17,4 +17,8 @@
       >
     </li>
   </ul>
+  <br />
+  <div class="flex list-none items-center justify-center gap-8">
```

Comment:
```suggestion
  <div class="flex list-none items-center justify-center">
```

not sure weather this is needed or not.

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question